utils/pron/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+
+include_directories( ${ENGINE_COMMON_INCLUDES} )
+
+set(pron_LIB_SRCS pron.cpp)
+
+add_library(pron STATIC ${pron_LIB_SRCS})
+
+target_link_libraries(pron messageqcpp loggingcpp)
+
+install(TARGETS pron DESTINATION ${ENGINE_LIBDIR} COMPONENT columnstore-engine)
+
+
```

Comment:
Please remove the extra linebreaks in this file.

Reply:
++